bootable-jar/WFLY-13305_Bootable_Jar.adoc

```diff
@@ -0,0 +1,505 @@
+be found in WildFly administration guide https://docs.wildfly.org/20/Admin_Guide.html#CLI_Recipes[CLI recipes chapter].
+
+In the context of Bootable JAR, the script does not need to contain commands to connect to the server or start an embedded server. 
+The Maven plugin handles that for you by starting an embedded server for each group of scripts.
+
+The plugin allows you to execute multiple groups of scripts with different CLI contexts. 
+A group of scripts and its configuration are defined in a ```cli-session``` composed of:
+
+* ```<script-files>```: the list of paths to script files .
+* ```properties-file```: (optional) a path to a properties file that contains java properties that scripts can reference (using the syntax ```${my.prop}```). 
+For example, a command that sets the public inet-address to the value of ```all.addresses``` system property looks like: ```/interface=public:write-attribute(name=inet-address,value=${all.addresses})```
+* ```resolve-expressions```: (optional) a boolean indicating if system properties or 
+expressions are resolved before sending the operation requests to the server. Value is ```true``` by default.
+
+All scripts present in a ```cli-session``` are executed within a single CLI execution. An embedded server is started for each defined ```cli-session```.
+
+NB: The scripts are executed in the order they are defined in the plugin configuration. 
+
+CLI configuration example:
+```
+<cli-sessions>
+  <cli-session>
+    <script-files>
+        <script>../scripts/script1.cli</script>
+    </script-files>
+    <!-- We want the env variables to be resolved during server execution -->
+    <resolve-expressions>false</resolve-expressions>
+  </cli-session>
+  <cli-session>
+    <script-files>
+        <script>../scripts/script2.cli</script>
+    </script-files>
+    <properties-file>../scripts/cli.properties</properties-file>
+    <!-- We want the properties to be resolved during CLI execution (not actually needed, this is the default behavior) -->
+    <resolve-expressions>true</resolve-expressions>
+  </cli-session>
+</cli-sessions>
+```
+
+===== Logging configuration
+
+During packaging the maven plugin generates a logging.properties file that reflects the server logging configuration. This ensures that the bootable JAR 
+will boot with a logging configuration in sync with the logging subsystem.
+
+IMPORTANT: Using a log4j appender as a `custom-handler` in the logging subsystem is not supported with the bootable JAR.
+           This only applies to custom handlers defined on the root of the logging subsystem. Logging profiles and
+           log4j configuration files located in your deployment will still work as expected.
+
+You can workaround this by supplying your own `logging.properties` and defining the path in the `boot-logging-config` maven plugin's configuration property.
```

Comment:
What exactly should be worked around?
Please, can you rephrase or rewrite this section so that a end user can understand how to configure the logging subsystem when using a bootable JAR, what is the standard use case (e.g. just use the generated `logging.properties` file) and something about custom scenarios? As usual an example would probably be useful here. WDYT?

Reply:
Yes, @jamezp will come-up with a workaround that we will use for documentation.

Reply:
Ok, thanks, let's have a link for that example in the proposal.

Reply:
I added a sentence saying that a log4j example is located in wildfly-jar-maven-plugin examples (will be).